src/@next/DataTable/TableHeader.tsx

```diff
@@ -0,0 +1,65 @@
+import React, { useState, useEffect } from 'react';
+import { Button } from '../Button';
+import { Icon } from '../Icon';
+import { SortDirection, TableTypography } from './DataTable';
+import { StyledTabledHeader } from './DataTableStyle';
+
+export interface TableHeaderProps
+  extends React.ThHTMLAttributes<HTMLTableColElement> {
+  title: string;
+  sortDirection?: SortDirection;
+  onSort?: (sortDirection: SortDirection) => void;
+}
+
+export const TableHeader = ({
+  sortDirection: sortDirectionProp,
+  title,
+  align = 'left',
+  onSort,
+}: TableHeaderProps) => {
+  const [sortDirection, setSortDirection] = useState(null);
```

Comment:
why did we use null instead of sortDirection for it's initial value since we ask for a default direction?

Reply:
because it'll be redundant with ` useEffect(() => {
    setSortDirection(sortDirectionProp);
  }, [sortDirectionProp]);`